README.MD

```diff
@@ -58,6 +58,15 @@ and background processing.
 | `SNS_TOPIC_CARD_PAYMENT_EVENTS_ARN`            | -       | ARN of the SNS topic for card payment events                                                               |
 | `SNS_TOPIC_CARD_PAYMENT_DISPUTE_EVENTS_ARN`    | -       | ARN of the SNS topic for card payment dispute events                                                       |
 
+### Expunge/Redact historic data configuration
+
+| Variable                                         | Default  | Purpose                                                                         |
+|--------------------------------------------------|----------|---------------------------------------------------------------------------------|
+| `EXPUNG_AND_REDACT_HISTORICAL_DATA_ENABLED`      | `false`  | Set to true to enable redacting PII from transactions and remove related events |
+| `EXPUNGE_OR_REDACT_DATA_OLDER_THAN_DAYS`         | `2555`   | Minimum age of transactions in days that need to be redacted/expunged           |
```

Comment:
Not a big issue, but is it worth putting "7 years" in the description? 2555 days is a big number.

Reply:
👍🏼 updated!